blocks/header/header.css

```diff
@@ -50,6 +50,40 @@
   width: 100%;
 }
 
+.header__link:any-link {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  color: var(--c-grey-4);
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-family: var(--ff-volvo-novum);
+  font-size: 14px;
```

Comment:
FYI: I can't use font variables because it use the `rem` units, and there is differnce in the document font size for pages after the redesign and pages that are not in the scope of the redesign.